experimental/instrumentation-xhr/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@grafana/faro-instrumentation-xhr",
+  "version": "1.0.0",
```

Comment:
All web-sdk package have the same version number. 
This to ensure to the best extend that we will not have issues with people mixing up different versions of faro packages.

Current version is  "1.1.2"